test/unit/node/api.js

```diff
@@ -14,7 +14,7 @@
 
 import {suite, test, assert} from '../../unit/unitTestRunner.js';
 
-suite('api.js', function() {
+suite('node-only: api.js', function() {
```

Comment:
Can't we just use the path since all the node related tests have `/node/` in their path?